ligo/src/registryDAO.mligo

```diff
@@ -169,11 +174,11 @@ let default_registry_DAO_full_storage (admin, token_address, a, b, s_max, c, d
     extra = Map.literal [
           ("registry" , Bytes.pack (Map.empty : registry));
           ("registry_affected" , Bytes.pack (Map.empty : registry_affected));
-          ("a" , Bytes.pack a); // frozen_scale_value
-          ("b" , Bytes.pack b); // frozen_extra_value
-          ("s_max" , Bytes.pack s_max); // max_proposal_size
-          ("c" , Bytes.pack c); // slash_scale_value
-          ("d" , Bytes.pack d); // slash_division_value
+          ("a" , Bytes.pack (Some (a))); // frozen_scale_value
+          ("b" , Bytes.pack (Some (b))); // frozen_extra_value
+          ("s_max" , Bytes.pack (Some (s_max))); // max_proposal_size
+          ("c" , Bytes.pack (Some (c))); // slash_scale_value
+          ("d" , Bytes.pack (Some (d))); // slash_division_value
```

Comment:
I don't understand why the need for this change, can you please clarify this?

Reply:
This is a misunderstanding on my part. I figured the configuration values need to be optional values in the storage, so the rest of the ligo code try to decode them as optional value. But here, I thought I pack them as non-optional values by mistake.

 But reading the spec again, it appears that they are optional only in config proposal, and in storage they are non-optional values. So I ll revert here and fix the unpacking code to expect non-optional values.